### DISCUSSION TITLE
Winter backend is not working properly

### DISCUSSION THREAD

**Discussion by jpassolli** [wintercms/winter#962] [Q&A]
My winter backend is having problems, it doesn't load plugins and the code editing pages are white, where there is a css file it's buggy...Does anyone know anything?

Pages don't load and I can't install plugins.

![Captura de Tela 2023-08-25 às 21 15 43](https://github.com/wintercms/winter/assets/127617505/3a971f86-4655-450d-a9ad-113188f82259)
![image](https://github.com/wintercms/winter/assets/127617505/61ddf8ee-cde8-4fd2-96d3-6c05239be8c3)

**Comment by bennothommo** [comment]
Hi @jpassolli, do you have any errors appearing in your Developer Tools in your browser?
